clients/ts/FunctionalTests/package-lock.json

```diff
@@ -16,21 +16,21 @@
     "@aspnet/signalr-protocol-msgpack": {
       "version": "file:../signalr-protocol-msgpack",
       "requires": {
-        "msgpack5": "^4.0.2"
```

Comment:
I suggest updating the version of npm you have (by running `npm install -g npm`) and run the build script again, then commit any changes to the `package-lock.json` file. These kind of changes can occur when NPM versions change.